apps/site/components/MDX/Details/index.tsx

```diff
@@ -0,0 +1,21 @@
+import type { FC, PropsWithChildren } from 'react';
+
+import styles from './index.module.css';
+
+type DetailsProps = {
+  summary: React.ReactNode;
```

Comment:
```suggestion
  summary: ReactNode;
```

Don't forget to import the type.